applications/StatisticsApplication/python_scripts/temporal_statistics_process.py

```diff
@@ -15,7 +15,17 @@ def Factory(settings, model):
         )
     return TemporalStatisticsProcess(model, settings["Parameters"])
 
+'''
+    Temporal Statistics process
 
+    This process calculates temporal statistics for given input variables in given container, and outputs to chosen variables
+    and chosen container.
+
+    This is compatible in OpenMP and MPI
+
+    Note: When this process is used with restarting, please don't use restarting start timestep and "statistics_start_point_control_value" time step
```

Comment:
I don't know the details but it would be nice if this can work in restarts without manual modifications ;)
Speaking from experience it is difficult keeping track of what needs to be changed where (for restarts)

Reply:
since this is regarding the starting point of time integration and restart starting point, There is a less probability both of them being the same point. So I figured, it might not be a bigger issue. That's why I left a note. :)

Thanks for the approval.